hare4/legacy_oracle.go

```diff
@@ -26,7 +26,7 @@ func (lg *legacyOracle) validate(msg *Message) grade {
 	if msg.Eligibility.Count == 0 {
 		return grade0
 	}
-	committee := int(lg.config.CommitteeFor(msg.Layer))
+	committee := int(lg.config.Committee)
```

Comment:
So my assumption was:
- this will get merged and the upgrade will happen after your changes that were needed for the committee upgrade. aka this will be shipped to mainnet after the committee upgrade already have happened. that's why I removed it (and the corresponding tests).

Reply:
Yes, the committee upgrade has landed. But we need to change it back eventually (do another upgrade). Do you plan to do this upgrade along with the switch to hare4? Besides, we might need to do committee upgrades in the future again. Keeping this code doesn't hurt and it might be useful. It can always be removed later after we stabilize the whole thing.

Reply:
OK